Bio/SearchIO/_model/hit.py

```diff
@@ -147,26 +147,32 @@ def __init__(self, hsps=(), id=None, query_id=None):
             self.append(hsp)
 
     def __repr__(self):
+        """Return Hit id, query id, and number of hsps."""
```

Comment:
Technically `__repr__` is not returning those values, but rather it returns a string representation of the object.